client/e2e/workflows/bceidbusiness/industry_user.spec.ts

```diff
@@ -138,60 +131,54 @@ test.describe("Test Workflow industry_user", () => {
     // 🛸 Navigate directly to the operator page (already tested navigating from the dashboard in the first test)
     const selectOperatorPage = new OperatorPOM(page);
     await selectOperatorPage.route();
+    // 🔍 Assert current URL
     await selectOperatorPage.urlIsCorrect();
-    await selectOperatorPage.linkAddOperator.click();
-    await expect(page.getByText(/Operator Information/i)).toBeVisible();
-
+    // 👉 Action create a new operator
+    await selectOperatorPage.clickAddOperator();
+    // 📷 Cheese!
     pageContent = page.locator("html");
     await happoPlaywright.screenshot(page, pageContent, {
       component: "Add a new operator",
       variant: "default",
     });
```

Comment:
I was looking through the Happo diffs and it looks like the majority of them are intended diffs due to various changes in the test. At the very top of the run is this diff which is problematic though, I think we need an assertion that the form is visible before taking a pic!

<img width="1072" alt="Screenshot 2024-04-05 at 11 54 53 AM" src="https://github.com/bcgov/cas-registration/assets/14259474/bf3805f5-22e7-4575-a352-af87ed9f2893">